src/tbb/task_group_context.cpp

```diff
@@ -65,11 +72,7 @@ void task_group_context_impl::destroy(d1::task_group_context& ctx) {
 #endif
     ctl->~cpu_ctl_env();
 
-    auto exception = ctx.my_exception.load(std::memory_order_relaxed);
-    if (exception) {
-        exception->destroy();
-    }
-    ITT_STACK_DESTROY(ctx.my_itt_caller);
```

Comment:
The ITT notification should be preserved.

Reply:
Done